static/elements/chromedash-featurelist.js

```diff
@@ -209,19 +209,25 @@ class ChromedashFeaturelist extends LitElement {
   }
 
   // Directly called from template/features.html
-  filter(val) {
+  filter(val, shouldPushState) {
+    this.searchEl.value = val;
+    const pushOrReplaceState = (
+      history &&
+          shouldPushState ? history.pushState.bind(history) :
+        history.replaceState.bind(history));
```

Comment:
Should this be this?
```
history && (shouldPushState ? history.pushState.bind(history) : history.replaceState).bind(history)
```

Reply:
Good catch. That is a logic error.  It would show up on any browser that did not have a history object.  I will fix.